src/libs/pois.js

```diff
@@ -9,7 +9,19 @@ export function toUrl(poi) {
   if (poi.id === 'geolocalisation' || poi.type === 'latlon') {
     return `latlon:${poi.latLon.lat.toFixed(5)}:${poi.latLon.lng.toFixed(5)}`;
   }
-  return `${poi.id}@${ExtendedString.slug(poi.name)}`;
+  return poi.name ? `${poi.id}@${ExtendedString.slug(poi.name)}` : poi.id;
+}
+
+export async function getInputValue(poi) {
```

Comment:
Can we define these functions as methods on parent class `Poi` ?

Reply:
this way, any inherited classes like `Bragi_poi` would benefit from the default implementation, and could override it

Reply:
Nope, the overall plan is to get rid of the classes completely, not complexify them :) Most of the functions in this file are already factorizations of behaviors that were scattered in all different POI classes, which is complex and increases special cases and such.
At term I'd like to manipulate simple flat structures with pure functions only.